projects/plugins/jetpack/_inc/client/components/auth-iframe/index.jsx

```diff
@@ -6,6 +6,8 @@ import { noop } from 'lodash';
 import PropTypes from 'prop-types';
 import React from 'react';
 import { __ } from '@wordpress/i18n';
+import { InPlaceConnection } from '@automattic/jetpack-connection';
+import { thirdPartyCookiesFallbackHelper } from '@automattic/jetpack-connection/helpers';
```

Comment:
Should we export `thirdPartyCookiesFallbackHelper` directly from `projects/js-packages/connection/index.jsx`, so we wouldn't have to rely on importing from a subfolder?

Reply:
I've been asking myself the same question, actually.
By exporting helpers from a separate file we logically separate them from the actual components.
On the other hand, that might be excessive, and having "Helper" in the name seems to be enough.